components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -47,18 +51,29 @@ export default function EditUserInformationModal({ children }: EditUserInformati
 
   //   TODO: Add error states for each field (post-mvp)
 
-  const successfulEditToast = useToast();
+  let loading = false;
 
-  const handleSubmit = (event: SyntheticEvent) => {
+  /**
+   * Handle edit submission
+   */
+  const handleSubmit = async (event: SyntheticEvent) => {
+    loading = true;
     event.preventDefault();
-    // TODO: Will be addressed in API hookup
-    onClose();
-
-    successfulEditToast({
-      render: () => (
-        <SuccessfulEditAlert>{"User's information has been edited."}</SuccessfulEditAlert>
-      ),
+    onSave({
```

Comment:
```suggestion
    await onSave({
```
Our `onSave` function is asynchronous so we would need to `await` for it to finish before we set `loading` to `false`